ser/data.py

```diff
@@ -0,0 +1,30 @@
+from torchvision import datasets, transforms
+from torch.utils.data import DataLoader
+from pathlib import Path
+
+PROJECT_ROOT = Path(__file__).parent.parent
+DATA_DIR = PROJECT_ROOT / "data"
+
+def prepare_data(batch_size):
+    """Prepare training and validation data loaders."""
+    # Torch transforms
+    ts = transforms.Compose(
```

Comment:
if the transforms become more complex or you need to configure them, it would be more readable to move them into their own file. However, this works well for these simpler code requirements!